packages/notifier/src/subscriber.js

```diff
@@ -46,8 +46,8 @@ const makeSubscriptionIterator = tailP => {
   return Far('SubscriptionIterator', {
     subscribe: () => makeSubscription(tailP),
     [Symbol.asyncIterator]: () => makeSubscriptionIterator(tailP),
-    next: () => {
-      const resultP = E.get(tailP).head;
+    next: async () => {
+      const resultP = await E.get(tailP).head;
```

Comment:
This one _might_ not be necessary, but it does seem problematic to eagerly to let an eager consumer (such as the new makeNotifierFromAsyncIterable) read an arbitrarily long sequence of results that might never materialize.